blas/unit_test/Test_Blas1_axpby.hpp

```diff
@@ -155,7 +160,8 @@ void impl_test_axpby_mv(int N, int K) {
 
   for (int i = 0; i < N; i++) {
     for (int j = 0; j < K; j++) {
-      EXPECT_NEAR_KK(a * h_x(i, j) + b * h_b_org_y(i, j), h_y(i, j), eps);
+      EXPECT_NEAR_KK(static_cast<ScalarB>(a * h_x(i, j) + b * h_org_y(i, j)),
+                     h_y(i, j), eps);
```

Comment:
Should ``eps`` be calculated from the A/B random bounds like in the non-MV case?